crates/pipeline_manager/src/db/mod.rs

```diff
@@ -246,87 +241,42 @@ pub(crate) struct ProjectDescr {
     pub schema: Option<String>,
 }
 
-/// Project configuration descriptor.
+/// PipelineDescr descriptor.
 #[derive(Serialize, ToSchema, Eq, PartialEq, Debug, Clone)]
-pub(crate) struct ConfigDescr {
-    pub config_id: ConfigId,
-    pub project_id: Option<ProjectId>,
-    pub pipeline: Option<PipelineDescr>,
+pub(crate) struct PipelineDescr {
+    pub pipeline_id: PipelineId,
+    pub program_id: Option<ProgramId>,
     pub version: Version,
     pub name: String,
     pub description: String,
     pub config: String,
     pub attached_connectors: Vec<AttachedConnector>,
+    pub port: u16,
+    pub shutdown: bool,
```

Comment:
Should shutdown be a boolean field of its own, as opposed to having a status field describe various states, one of which is "shutdown"? I anticipate we will have states like Pending, Compiling, Deploying, Running, Error, and Shutdown.

Reply:
Agreed this is one of the remaining issues; have a proper pipeline status in the field, I will make a separate PR for it so we can get this in sooner (the status field needs more changes in the UI because right now there is a bunch of code tracking the status in the UI on the client which can all go away then)